vendor/example/windsensor.yaml

```diff
@@ -98,13 +98,23 @@ keySecurity: none
 # Product and data sheet URLs (optional)
 productURL: https://example.org/wind-sensor
 dataSheetURL: https://example.org/wind-sensor/datasheet.pdf
+resellerURLs:
+  - name: 'Reseller 1'
+    region: Global
+    url: https://example.org/reseller1
+  - name: 'Reseller 2'
+    region: Europe
```

Comment:
See https://github.com/lorawan-schema/draft-devices/pull/7#discussion_r528557978; "Europe" is not a market